homeassistant/components/smartthings/light.py

```diff
@@ -203,9 +211,11 @@ def color_mode(self) -> ColorMode:
             return list(self._attr_supported_color_modes)[0]
 
         # The light supports hs + color temp, determine which one it is
-        if self._attr_hs_color and self._attr_hs_color[1]:
+        if self._last_color_mode is Capability.COLOR_CONTROL:
             return ColorMode.HS
-        return ColorMode.COLOR_TEMP
+        if self._last_color_mode is Capability.COLOR_TEMPERATURE:
+            return ColorMode.COLOR_TEMP
+        return ColorMode.ONOFF
```

Comment:
One concern I have with this is, what is the most correct start up logic? I now return `ONOFF`. I don't see any other way to assume a correct last state. Would maybe be using a restore function be a good middle ground here?